docs/user_docs/kubeblocks-for-mysql/configuration/configuration.md

```diff
@@ -285,6 +287,45 @@ Static parameter reconfiguring requires restarting the pod. The following exampl
    1 row in set (0.09 sec)
    ```
 
+## Reconfigure parameters with edit-config
+
+For your convenience, kbcli offers a tool `edit-config` to help you to visually edit configurations.
+
+For Linux and macOS, you can edit configuration files by vi and for Windows, you can edit files on notepad.
+
+The following steps take configuring MySQL Standalone as an example.
+
+1. Edit the configuration file.
+
+   ```bash
+   kbcli cluster edit-config mysql-cluster
```

Comment:
Now, apecloud has multiple config templates, and need to specify the --configspec， e.g: kbcli cluster edit-config mysql-cluster --config-spec=mysql-consensusset-config